templates/deployment-client.yaml

```diff
@@ -45,6 +45,10 @@ spec:
           {{- if .Values.server }}
             - name: OPAL_SERVER_URL
               value: {{ printf "http://%s:%v" (include "opal.serverName" .) .Values.server.port | quote }}
+            {{- if not (or (.Values.server.dataConfigSources.external_source_url) (len .Values.server.dataConfigSources.config.entries)) }}
```

Comment:
If I get this right, it means that *both* - 
1. external_source_url not exist
2. **AND** the len of config.entries is 0. 
right ?

Reply:
Right... 
So the client won't get any data updates...